x-pack/plugins/apm/server/lib/alerts/register_transaction_duration_alert_type.ts

```diff
@@ -107,33 +86,17 @@ export function registerTransactionDurationAlertType({
                     },
                   },
                 },
-                {
-                  term: {
-                    [PROCESSOR_EVENT]: 'transaction',
-                  },
-                },
-                {
-                  term: {
-                    [SERVICE_NAME]: alertParams.serviceName,
-                  },
-                },
-                {
-                  term: {
-                    [TRANSACTION_TYPE]: alertParams.transactionType,
-                  },
-                },
-                ...environmentTerm,
+                { term: { [PROCESSOR_EVENT]: 'transaction' } },
```

Comment:
Same comment here as above.

Reply:
Yes! I had actually fixed it after your previous comment. Forgot to push...